extensions/amp-analytics/0.1/vendors.js

```diff
@@ -110,6 +111,8 @@ const DEFAULT_CONFIG = jsonLiteral({
     'ampdocHostname': 'AMPDOC_HOSTNAME',
     'ampdocUrl': 'AMPDOC_URL',
     'ampGeo': 'AMP_GEO',
+    'ampUserLocation': 'AMP_USER_LOCATION',
+    'ampUserLocationPoll': 'AMP_USER_LOCATION_POLL',
```

Comment:
Why are these getting added?

Reply:
I must have taken an older version of the file.
Fixed! Thanks!